src/database/queries/proposals/selectWithPagination.sql

```diff
@@ -4,5 +4,15 @@ SELECT p.id AS "id",
   p.opportunity_id AS "opportunityId",
   p.created_at AS "createdAt"
 FROM proposals p
+  LEFT JOIN proposal_versions pv ON pv.proposal_id = p.id
```

Comment:
I don't know if this has a performance impact we need to worry about, but these joins are unnecessary if `:search` is undefined.  I wasn't quite sure how to have the joins themselves only occur in a case clause.

Reply:
Understood -- I think this will be something to explore in more depth when we address #314